.gitignore

```diff
@@ -85,4 +86,4 @@ cmd/test-driver
 *.snap
 
 # Image files
-*.img
\ No newline at end of file
+*.img
```

Comment:
Unrelated changes should be in a separate PR.

Reply:
Thanks for the feedback! The commit has already been merged so I'll drop it when I squash the other commits (I included it because it hadn't been merged when I opened the PR)